dask/array/reductions.py

```diff
@@ -92,7 +93,8 @@ def partial_reduce(func, x, split_every, keepdims=False, dtype=None, name=None):
 
     >>> partial_reduce(np.min, x, {0: 1, 2: 3})    # doctest: +SKIP
     """
-    name = name or 'p_reduce-' + tokenize(func, x, split_every, keepdims, dtype)
+    name = name or funcname(func) + '-' + tokenize(func, x, split_every,
```

Comment:
I'm confused, how did we pass down the level information previously?  This looks like we're just replacing `p_reduce` with `funcname(func)`.